### PR TITLE
doc: Add contact info

### PR DESCRIPTION
**Pull request by anders-kolstad**
Just a test

### CODE REVIEW

**Comment by anders-kolstad** [comment]
This looks good Anders!

**Comment by DrMattG** [comment]
This is a bad idea!

File: README.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 This repository is for documenting the design and calculation of indicators for ecosystem condition in Norway.
 
+Contact: Anders L. Kolstad (anders.kolstad@nina.no)
```
Comment (anders-kolstad) [inline README.md:5]: This is a good change

Comment (anders-kolstad) [reply README.md:5]: Perhaps add Matt and Hanno?

**Review by anders-kolstad — COMMENTED** [review]
Approved (by myself)

**Pull request merged**